mParticleCore.brs

```diff
@@ -424,6 +424,13 @@ function mParticleConstants() as object
             session.streamType = streamType
             session.mediaSessionId = CreateObject("roDeviceInfo").GetRandomUUID()
             session.currentPlayheadPosition = 0
+            session.mediaContentComplete = false
+            session.mediaContentTimeSpent = 0
+            session.mediaSessionSegmentTotal = 0
+            session.mediaSessionAdTotal = 0
+            session.mediaTotalAdTimeSpent = 0
+            session.mediaAdTimeSpentRate = 0.0
+            session.mediaSessionAdObjects = createObject("roArray", 0, true)
```

Comment:
Is this a typo? Shouldn't it be `CreateObject`?

Reply:
yep good catch